<?xml version="1.0"?>
<!-- 
  Wrapper launch file for the i5 specific motion streaming interface node.
-->
<launch>
  <!-- IP of robot (or PC running simulation) -->
  <arg name="robot_ip" doc="IP of controller" />


  <!-- put them on the parameter server -->
  <param name="robot_ip_address" type="str" value="$(arg robot_ip)" />

  <!-- load the correct version of the motion streaming node -->
  <node name="motion_streaming_interface"
    pkg="i5_driver" type="motion_streaming_interface" />
    
  <node pkg="industrial_robot_client" type="joint_trajectory_action" 
	name="joint_trajectory_action" />
</launch>